hermes/probe/create/create_file_test.go

```diff
@@ -0,0 +1,140 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	metricpb "github.com/google/cloudprober/metrics/proto"
+	probepb "github.com/googleinterns/step224-2020/config/proto"
+	"github.com/googleinterns/step224-2020/hermes/probe"
+	"github.com/googleinterns/step224-2020/hermes/probe/metrics"
+	"github.com/googleinterns/step224-2020/hermes/probe/mock"
+	journalpb "github.com/googleinterns/step224-2020/hermes/proto"
+)
+
+func TestFileName(t *testing.T) {
+	tests := []struct {
+		file     RandomHermesFile
+		wantName string
+		wantErr  string
```

Comment:
With the idea of having a `ProbeError` as the return value for CreateFile, you could change this line to `wantStatus  metrics.ProbeError`. Then your tests instead of having the error string could have something like `metrics.OutOfRange`, `metrics.Success`, or `metrics.InvalidArgument`

Reply:
This will be left as a TODO in probe.go